package-lock.json

```diff
@@ -1206,38 +1794,7 @@
     "inherits": {
       "version": "2.0.3",
       "resolved": "https://registry.npmjs.org/inherits/-/inherits-2.0.3.tgz",
-      "integrity": "sha512-x00IRNXNy63jwGkJmzPigoySHbaqpNuzKbBOmzK+g2OdZpQ9w+sxCN+VSB3ja7IAge2OP2qpfxTjeNcyjmW1uw=="
-    },
-    "ioredis": {
```

Comment:
This downgrade seems to be wrong, but apparently the previous commit (revert) was not complete https://github.com/overleaf/real-time/commit/90742cd60d39ba06d46c2ca5fe7042f14f23105c .
cc @mserranom did you revert parts of a previous commit or ran `bin/run real-time npm install redis-sharelatex@1.0.12` to produce the commit?

Reply:
I did it via `npm`, but not sure if I did install it from the previous commit, or did update the dependencies in a different commit, then committed the resulting lockfile on top of it.